From ab0228c3d6ceba20cf89ceb1b16b7e314aaaf989 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Tue, 7 Aug 2018 10:38:33 +0200
Subject: [PATCH] net_setup_link: allow renaming interfaces that were renamed
 previously

---
 src/udev/net/link-config.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/udev/net/link-config.c b/src/udev/net/link-config.c
index cec4f4f779..5113586457 100644
--- a/src/udev/net/link-config.c
+++ b/src/udev/net/link-config.c
@@ -306,7 +306,6 @@ static bool should_rename(struct udev_device *device, bool respect_predictable)
 
         switch (type) {
         case NET_NAME_USER:
-        case NET_NAME_RENAMED:
                 /* these were already named by userspace, do not touch again */
                 return false;
         case NET_NAME_PREDICTABLE:
